modules/installation-disk-partitioning.adoc

```diff
@@ -24,6 +24,11 @@ officially supported for making `/var` or a subdirectory of `/var`, such as `/va
 +
 [IMPORTANT]
 ====
+For disk sizes larger than 100G, and especially disk sizes larger than 1TB, it is strongly recommended to create a separate `/var` partition. See "Creating a separate `/var` partition" for more information.
```

Comment:
- Should "100G" be "100GB?"
- Shouldn't recommend thing: https://redhat-documentation.github.io/supplementary-style-guide/#_r
<img width="985" alt="image" src="https://user-images.githubusercontent.com/83772007/168334842-dc913c8d-c333-48a5-ad05-d2adca6615e9.png">

Reply:
@nalhadef good catch! What do you think about just omitting that phrase? so something like:

```suggestion
For disk sizes larger than 100G, and especially disk sizes larger than 1TB, create a separate `/var` partition. See "Creating a separate `/var` partition" for more information.
```

Reply:
Thanks, @jldohmann . Yep, perfect solution!!!